client/src/js/services/FinancialService.js

```diff
@@ -0,0 +1,39 @@
+angular.module('bhima.services')
+.service('FinancialService', FinancialService);
+
+FinancialService.$inject = [ '$http', 'util' ];
+
+function FinancialService ($http, util) {
+  var service = this;
+
+  service.readCostCenter = readCostCenter;
```

Comment:
@lomamech, I still do not understand the difference between these two methods.  Could you include comments describing what each method does?